src/main/java/ua/softserveinc/tc/dto/MonthlyEventDto.java

```diff
@@ -17,24 +17,26 @@
 
     public MonthlyEventDto() {
     }
-    public static MonthlyEventDto getMonthlyEventDto(List<Event> listOfRecurrentEvent, int[] daysOfTheMonth) {
+
+    public MonthlyEventDto(Event event) {
+        super(event);
+    }
+
+    public static MonthlyEventDto getMonthlyEventDto(List<Event> listOfRecurrentEvent,
+                                                     int[] daysOfTheMonth) {
         Event startDay = listOfRecurrentEvent.get(0);
         Event endDay = listOfRecurrentEvent.get(listOfRecurrentEvent.size() - 1);
         startDay.setEndTime(endDay.getEndTime());
         MonthlyEventDto monthlyEventDto = new MonthlyEventDto(startDay);
-        monthlyEventDto.setDaysOfMonth(daysOfTheMonth);
+        monthlyEventDto.setDaysOfTheMonth(daysOfTheMonth);
         return monthlyEventDto;
     }
 
-    public MonthlyEventDto(Event event) {
-        super(event);
-    }
-
-    public void setDaysOfMonth(int[] daysOfTheMonth) {
+    public void setDaysOfTheMonth(int[] daysOfTheMonth) {
         this.daysOfTheMonth = daysOfTheMonth;
     }
 
-    public int[] getDaysOfMonth() {
+    public int[] getDaysOfTheMonth() {
```

Comment:
be carefull with merging this one pls, i've made some changes, need it for demo))